frontend/src/PlanListContainer.css

```diff
@@ -7,6 +7,6 @@
     border-radius: 8px;
     box-sizing: border-box;
     max-width: 400px;
-    margin: 0 0 100px 0;
+    /*margin: 0 0 100px 0;*/
```

Comment:
Do you still need this margin? If not, I would delete it instead of leaving it in as a comment